Ubika/ubika-cloud-protector/_meta/manifest.yml

```diff
@@ -0,0 +1,9 @@
+uuid: d0383e87-e054-4a21-8a2c-6a89635d8615
+name: Ubika Cloud Protector
```

Comment:
```suggestion
name: Ubika Cloud Protector [BETA]
```

Reply:
>A logo is missing from the format. Can you add one?

Sure. Need to check the `utils/checks` script as well, then